src/api/spec/support/files/service

```diff
@@ -0,0 +1,14 @@
+<services>
```

Comment:
I would rename the file to broken service or something! It is on a first glance not clear that this is broken! Also is it really necessary to have a full file for this? Wouldn't it be enough to just have ``foobar`` as content for a broken service?